src/main/webapp/app/overview/exercise-details/programming-exercise-student-ide-actions.component.ts

```diff
@@ -32,7 +32,7 @@ export class ProgrammingExerciseStudentIdeActionsComponent implements OnInit {
     @Input() @HostBinding('class.col') equalColumns = true;
     @Input() @HostBinding('class.col-auto') smallColumns = false;
 
-    @Input() exercise: Exercise;
+    @Input() exercise: ProgrammingExercise;
```

Comment:
I believe the import above can now be removed (as github already detected), I can't add the review in the import statement for some reason (maybe one can only add it to changed lines).

Reply:
removed the import